libs/barista-components/radial-chart/src/radial-chart.ts

```diff
@@ -55,6 +57,15 @@ import {
   getSum,
 } from './utils/radial-chart-utils';
 
+export type DtChartHoverData = {
```

Comment:
Please rename the type so it will not lead to confusions. 
```suggestion
export type DtRadialChartHoverData = {
```